sphinx/source/docs/user_guide/objects.rst

```diff
@@ -268,6 +268,9 @@ region by left-dragging a mouse, or dragging a finger across the plot area.
 The box select tool may be configured to select across only one dimension by
 setting the ``dimension`` property to ``width`` or ``height``.
 
+.. note:: To make a multiple selection, you have to press the SHIFT key and
+to clear the selection you have to press ESC key
```

Comment:
Sphinx is picky, I don't think this will render correctly as an admonition. I know write all admonitions like this:
```
.. note:: 
    To make a multiple selection, you have to press the SHIFT key and
    to clear the selection you have to press ESC key
```
That is, starting indented on the line below